src/plots/xyPlot.ts

```diff
@@ -432,13 +433,18 @@ export class XYPlot<X, Y> extends Plot {
 
   /**
    * _invertPixelPoint converts a point in pixel coordinates to a point in data coordinates
+   * (if neither scale is Category)
    * @param {Point} point Representation of the point in pixel coordinates
    * @return {Point} Returns the point represented in data coordinates
    */
   protected _invertPixelPoint(point: Point): Point {
     const xScale = this.x();
     const yScale = this.y();
 
+    if (xScale.scale instanceof Category || yScale.scale instanceof Category) {
+      return point;
```

Comment:
nit: would be good to make this more general; seems a bit special casey. Any ideas?